crates/nargo/src/cli/mod.rs

```diff
@@ -33,8 +32,12 @@ mod prove_cmd;
 mod test_cmd;
 mod verify_cmd;
 
-const SHORT_GIT_HASH: &str = git_version!(prefix = "git:");
-const VERSION_STRING: &str = formatcp!("{} ({})", env!("CARGO_PKG_VERSION"), SHORT_GIT_HASH);
+const GIT_HASH: &str = env!("GIT_COMMIT");
+const IS_DIRTY: &str = env!("GIT_DIRTY");
+const CARGO_PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+static VERSION_STRING: &str =
+    formatcp!("{} (git version hash: {}, is dirty: {})", CARGO_PKG_VERSION, GIT_HASH, IS_DIRTY);
```

Comment:
```suggestion
    formatcp!("{} (git: {}, dirty: {})", CARGO_PKG_VERSION, GIT_HASH, IS_DIRTY);
```

My preference is to avoid spaces in the "keys" here but either way works.

Reply:
Unsure about this? This conveys information to the user in consumable format, if the same info is useful in machine format, we can have it output json with flag.